target/linux/sunxi/base-files/etc/board.d/02_network

```diff
@@ -7,19 +7,22 @@
 
 board_config_update
 
-case "$(board_name)" in
-"olimex,a20-olinuxino-micro")
-	ucidef_set_interface_lan 'wlan0'
+case $(board_name) in
+friendlyarm,nanopi-r1)
+	ucidef_set_interfaces_lan_wan "eth1" "eth0"
 	;;
-"lamobo,lamobo-r1")
+lamobo,lamobo-r1)
 	ucidef_add_switch "switch0" \
 		"4:lan:1" "0:lan:2" "1:lan:3" "2:lan:4" "3:wan" "8@eth0"
 	;;
-"xunlong,orangepi-r1")
+olimex,a20-olinuxino-micro)
+	ucidef_set_interface_lan "wlan0"
+	;;
+xunlong,orangepi-r1)
 	ucidef_set_interfaces_lan_wan "eth0" "eth1"
 	;;
 *)
-	ucidef_set_interface_lan 'eth0'
+	ucidef_set_interface_lan "eth0"
```

Comment:
Please rebase your changes onto recent master, so this file only shows your changes.